README.md

```diff
@@ -59,18 +59,29 @@ If required, Palisade allows an organisation to use centralised services which e
 ![Palisade Overview](doc/img/Palisade_overview.jpg)
 
 
+### Prerequisites
+1. git
+1. maven
+1. to run any of the examples: docker
+1. if running the docker based examples: docker-compose
+
```

Comment:
This is slightly unclear, I need Docker to run any examples, but to run docker examples I need docker-compose. So therefore I always need Docker-compose yes?